Add Web screenshots. (#3259)
beeware/toga: changes/3259.doc.rst

> Added Web screenshots for the Button, Label, ProgressBar, Switch, TextInput widgets on their respective documentation pages.

Reviewer: The voice/tense for this note is something you'd see in a release note:
```suggestion
Widget screenshots have been added for the Web backend.
```
